### ISSUE TITLE
Font rendering issue; characters appear at wrong size, position

### ISSUE THREAD

**Issue by tuscen** [microsoft/terminal#4367]
# Environment

```none
Windows build number: 10.0.18363.0
Windows Terminal version (if applicable): 0.8.10091.0

Font: JetBrains Mono
```

# Steps to reproduce

Sometimes font doesn't render properly: some characters are too tall or too short. Sometimes it's numbers, sometimes it's capital case letters. Regular conhost window or ConEmu don't have this problem. 

# Expected behavior

Font characters render properly.

ConEmu example. Everything seems to render properly.

<img width="396" alt="conemu" src="https://user-images.githubusercontent.com/1078199/73177350-dade1300-411f-11ea-822b-20eedbc62c46.png">
<img width="130" alt="conemu" src="https://user-images.githubusercontent.com/1078199/73178851-280fb400-4123-11ea-84c8-d08f6742d114.png">

# Actual behavior

Some characters are too tall or too short. You can see it in capital case letters.

<img width="287" alt="Аннотация 2020-01-27 154648" src="https://user-images.githubusercontent.com/1078199/73176761-52ab3e00-411e-11ea-9148-f3accb225661.png">
<img width="122" alt="terminal" src="https://user-images.githubusercontent.com/1078199/73178752-f8f94280-4122-11ea-86d0-99aeb5504e55.png">

**Comment by zadjii-msft** [comment]
This _might_ be related to #610, but this seems subtle enough that it might not be. I'm going to leave this open to make sure we address this particular issue.

**Comment by hnakamur** [comment]
I have the same issue for Japanese characters.
I created a gist to reproduce the problem https://gist.github.com/hnakamur/2a123ce7bdc0d7e5070218c258f990a8
I also added some screenshots there.

You can run the following command in a Ubuntu (Windows Subsystem for Linux) shell.
```
curl -sS https://gist.githubusercontent.com/hnakamur/2a123ce7bdc0d7e5070218c258f990a8/raw/b3cd604b6f8b0781cd51758afa3e1dd264e7c6d7/swap-myth.txt
```

**Comment by zadjii-msft** [comment]
I know @miniksa was mucking with the baselines in #5345, maybe this is related?

**Comment by tuscen** [comment]
Looks like it was fixed. I do use a 4K display with 150% scaling. It might be that #5345 fixed my issue.

![1](https://user-images.githubusercontent.com/1078199/79254073-dc041e00-7e8c-11ea-884a-6022a89346d3.jpg)
![2](https://user-images.githubusercontent.com/1078199/79254076-dc9cb480-7e8c-11ea-97e8-5146c08ead35.jpg)

**Comment by zadjii-msft** [comment]
@tuscen that's _excellent_ to hear. I'd usually love to have an actual commit to link this to for future people that run into this issue, but we've changed so much about our rendering stack since 0.8 that it'd be hard to pin this down exactly. 

Thanks for following up though!

**Comment by hnakamur** [comment]
Sorry for late reporting, but my issue was also resolved since version: 0.10.781.0.
Thanks for a great product!
